src/lib/utils/strong_type.h

```diff
@@ -118,6 +118,10 @@ class Container_Strong_Adapter_Base : public Strong_Base<T> {
       }
 
       decltype(auto) operator[](size_type i) noexcept(noexcept(this->get().operator[](i))) { return this->get()[i]; }
+
+      decltype(auto) at(size_type i) const { return this->get().at(i); }
+
+      decltype(auto) at(size_type i) { return this->get().at(i); }
```

Comment:
Fine to add these, but perhaps in the `bitvector` strong type adapter. This adapter is based on the `concepts::container` concept which does not require the underlying type to have `at()` (IIRC).

Reply:
Or do you need them for strong types on `std::vector<>`?

Reply:
Yeah, we were about to remove it, but your eyes were to fast :P

Reply:
The `.at` works for bitvectors but not for normal vectors. We thought about adding a Strong adapter for contiguous containers that support `.at`. This could be a good task for our template magician. What do you think?

Reply:
🪄